hack/aks/Makefile

```diff
@@ -98,6 +120,7 @@ up: swift-up ## Alias to swift-up
 
 
 nodesubnet-byocni-nokubeproxy-up: rg-up overlay-net-up ## Brings up an NodeSubnet BYO CNI cluster without kube-proxy
+	@$(MAKE) public-ipv4
```

Comment:
will this rerun the public ip creation? if so, should we have the ip creation as a global step so it provisions just once?

Reply:
yes, this reruns public ip creation, but since the Cilium pipelines do cluster creation in parallel, we need to provision different public ips